### ISSUE TITLE
Prisma takes a lot to connect [mongodb]

### ISSUE THREAD

**Issue by MARCROCK22** [prisma/prisma#11938]
### Bug description

While using "prisma.user.findFirst" takes a lot of time in the first request

### How to reproduce

1. Connect with @prisma/client [mongodb]
2. Send a request
![imagen](https://user-images.githubusercontent.com/57925328/154866920-3bca1661-d3d8-484c-a6ea-c2eea9eea9d0.png)

### Prisma information

```ts
import prisma from './prismaclient.js';

async function main() {
    prisma.$on('beforeExit', async () => {
        console.log('disconnecting from database');
        await prisma.$disconnect();
        console.log('disconnected from database');
    });
    console.time('query1');
    await prisma.user.findFirst();
    console.timeEnd('query1');
    console.time('query2');
    await prisma.user.findFirst();
    console.timeEnd('query2');
}

main();
```
prismaclient.ts
```ts
import __prisma from '@prisma/client';

const prisma = new __prisma.PrismaClient();

export default prisma;
```
schema.prisma
```
// This is your Prisma schema file,
// learn more about it in the docs: https://pris.ly/d/prisma-schema

datasource db {

  provider = "mongodb"

  url = env("DATABASE_URL")

}

generator client {

  provider = "prisma-client-js"

  previewFeatures = ["mongodb"]

}

model User {

  id String @id @default(dbgenerated()) @map("_id") @db.ObjectId

  email String @unique

  name String?

}
```
### Environment & setup

- OS: Windows
- Database: mongodb
- Node.js version: v16.10.0

### Prisma Version

```
prisma                  : 3.9.2
@prisma/client          : 3.9.2
Current platform        : windows
```

**Comment by janpio** [comment]
This will be fixed in 3.10.0. 
You can try with the current `dev` version: `npm install prisma@dev @prisma/client@dev` and should already get much better results.

**Comment by janpio** [comment]
This should now be fixed. Can you confirm?
